src/emails/StorybookEmailRenderer.tsx

```diff
@@ -31,22 +32,71 @@ export const StorybookEmailRenderer = (props: Props) => {
         background-color: #1e293b !important;
         color: white !important;
       }
+
+      .wrapper {
+        display: flex;
+        gap: 10px;
+      }
+
+      .badge {
+        align-self: flex-start;
+        display: inline-block;
+        font-family: monospace;
+        background-color: #eee;
+        border-radius: 3px;
+        text-transform: uppercase;
+        padding: 3px 5px;
+        color: #444;
+      }
+
+      .wrapper .styled {
+        flex: 2 0 auto;
+        width: 66%;
+        padding: 20px;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        gap: 20px;
+      }
+      .wrapper .plaintext {
+        flex: 1 0 auto;
+        width: 33%;
+        font-family: monospace;
+        border-inline-start: 1px solid #eee;
+        padding: 20px;
+        padding-inline-start: calc(10px + 20px);
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        gap: 20px;
+      }
+
+      .plaintext pre {
+        white-space: pre-wrap;
+      }
   `}</style>
-      <div
-        dangerouslySetInnerHTML={{
-          __html:
-            renderResult.errors.length > 0
-              ? `
-                <h1>MJML rendering errors:</h1>
-                <ul style="font-family: monospace;">${renderResult.errors.map((error) => `<li>${error.message}</li>`).join("\n")}</ul>
-                <hr/>
-                ${renderResult.html}
-                `
-              : renderResult.html,
-        }}
-        className={props.emulateDarkMode ? "dark-mode-enforced" : ""}
-      />
-    </>
+      <section className="styled">
+        <div className="badge">HTML</div>
+        <div
+          dangerouslySetInnerHTML={{
+            __html:
+              renderResult.errors.length > 0
+                ? `
+                  <h1>MJML rendering errors:</h1>
+                  <ul style="font-family: monospace;">${renderResult.errors.map((error) => `<li>${error.message}</li>`).join("\n")}</ul>
+                  <hr/>
+                  ${renderResult.html}
+                  `
+                : renderResult.html,
+          }}
+          className={props.emulateDarkMode ? "dark-mode-enforced" : ""}
+        />
+      </section>
+      <section className="plaintext">
+        <div className="badge">Plaintext</div>
+        <pre>{props.plainTextVersion ?? "Not set"}</pre>
```

Comment:
Is it intended that we are showing the string and not the rendered HTML here?

```suggestion
        {props.plainTextVersion ? (
          <div
            dangerouslySetInnerHTML={{
              __html: props.plainTextVersion,
            }}
            className={props.emulateDarkMode ? "dark-mode-enforced" : ""}
          />
        ) : (
          "Not set"
        )}
```

Reply:
@flozia I'm not sure I follow - the plain text version is an alternative to the HTML version, i.e. there shouldn't be HTML in there?

Reply:
@Vinnl In that case I misunderstood. I thought the idea was that is was plain text when looking at the [FXA email story](https://storage.googleapis.com/mozilla-storybooks-fxa/commits/d06a74b249ebfe979642fdc94f4086a3ed031312/fxa-auth-server/index.html?path=/story/subplat-emails-layout--layout-no-product-with-brand-messaging): There are still links in there — how would those be handled?

Reply:
Ha actually, I think that's a mistake on their side? Generally, it'll be up to the mail client to make spelled-out URLs clickable, or to the user to copy-paste it.

Reply:
Makes sense!